symfony/plugins/orangehrmTimePlugin/test/Dao/CustomerDaoTest.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+namespace OrangeHRM\Tests\Time\Dao;
+
+
+use Exception;
+use OrangeHRM\Entity\Customer;
+use OrangeHRM\Tests\Util\KernelTestCase;
+use OrangeHRM\Time\Dao\CustomerDao;
+use OrangeHRM\Time\Dto\CustomerSearchFilterParams;
+
+class CustomerDaoTest extends KernelTestCase
+{
+
+    private CustomerDao $customerDao;
+    protected string $fixture;
+
+    /**
+     * Set up method
+     * @throws Exception
+     */
+    protected function setUp(): void
+    {
+        $this->customerDao = new CustomerDao();
```

Comment:
add `TestDataService::truncateTables([Customer::class]);` this also

Reply:
done